<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BYOD-NIVA" failures="1" tests="1" name="Command line test" time="11.1" errors="0" timestamp="2022-03-20T20:38:38 IST">
  <testcase classname="com.learautomation.testcases.TestCaseWithoutFramework" name="test1" time="11.066">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Something went wrong...] but found [Something went wrong...M]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Something went wrong...] but found [Something went wrong...M]
at com.learautomation.testcases.TestCaseWithoutFramework.test1(TestCaseWithoutFramework.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- Command line test -->
